### PR TITLE
[DO NOT MERGE, DRAFT] Bhavikmfujikosu/eval sample geval

### PR DESCRIPTION
**Pull request by fujikosu**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/flows/evaluation/eval-summarization/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Summarization Evaluation
+
+This flow implements a reference-free automatic abstractive summarization evaluation across four dimensions: fluency, coherence, consistency, relevance. Each dimension uses a prompt to score a generated summary against the source document from which it was generated. Each dimension's prompt has been meta-evaluated against the [SummEval benchmark](https://arxiv.org/abs/2007.12626) and produces state-of-the-art scores in terms of correlation to expert human judgements.
+
+## Introduction
+
+Provide a detailed description of the flow, including its components, inputs, outputs, and any dependencies. Explain how the flow works and what problem it solves. This section should give users a clear understanding of the flow's functionality and how it can be used. -- TODO remove this when done
```
Comment (wangchao1230) [inline examples/flows/evaluation/eval-summarization/README.md:7]: Mark on this TODO

File: examples/flows/evaluation/eval-summarization/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Troubleshooting
+
+If there are any known issues or troubleshooting tips related to the flow, include them in this section. Provide solutions or workarounds for common problems that users may encounter. This will help users troubleshoot issues on their own and reduce the need for support.   -- TODO remove this when done, or remove this section if not needed
+
+## Contact
+
+If you have any questions or issues related to this flow, please reach out to either:
+
+- Bhavik Maneck [[Email](mailto:bhavikmaneck@microsoft.com) | [Linked-In](https://www.linkedin.com/in/bhavik-maneck/)]
```
Comment (wangchao1230) [inline examples/flows/evaluation/eval-summarization/README.md:105]: do you have a team alias email to put here?

Comment (fujikosu) [reply examples/flows/evaluation/eval-summarization/README.md:105]: we don't have team alias email

File: examples/flows/evaluation/eval-summarization/README.md (hunk 1 above)
Comment (fujikosu) [reply examples/flows/evaluation/eval-summarization/README.md:7]: thanks for your review! yeah the purpose of this PR was to test CI pipeline early actually. That's why we put [DO NOT MERGE, DRAFT] in the titile and kept multiple TODOs in readme we are still working on. It turns out forked repo's CI pipeline didn't have enough permission to run CI successfully so based on PG's recommendation, we moved to promptflow repo's branch and new PR is here. I'm closing this PR now as all the updates will come to that branch instead

https://github.com/microsoft/promptflow/pull/2037

**Pull request closed without merging**